example/plugins/c-api/lifecycle_plugin/lifecycle_plugin.c

```diff
@@ -49,6 +49,10 @@ CallbackHandler(TSCont this, TSEvent id, void *data)
   case TS_EVENT_LIFECYCLE_MSG: {
     TSPluginMsg *msg = (TSPluginMsg *)data;
     TSDebug(PLUGIN_NAME, "Message to '%s' - %zu bytes of data", msg->tag, msg->data_size);
+    if (msg->data_size == 0) {
```

Comment:
Isn't this clear from the message on line 51?

Reply:
yes, just want to make a point as this is an example plugin.